### DISCUSSION TITLE
GitHub連携でGistのread/write権限は必要か

### DISCUSSION THREAD

**Discussion by tenmyo** [increments/qiita-discussions#923] [要望・アイデア・不具合]
QiitaからGitHubへの連携をしています。連携時のPermissionで、Gistのread/write accessを求められます。
以前はGist連携がありましたが、下記お知らせのとおり2019年1月8日にてGist連携は廃止されたと認識しています。
なるべくPermissionは絞っていただきたいです。現在においてもGistのread/write access Permissionは必要でしょうか？

[記事投稿時におけるGist連携の提供終了のお知らせ - Qiita Blog](https://blog.qiita.com/close-gist-coordination/)

**Comment by kimkim0814** [comment]
@tenmyo 
ご報告ありがとうございます🙏

おっしゃる通り、QiitaのGitHub連携において、Gistのread/writeアクセス権限は現在不要です。

こちらについては、不要なGistのアクセス権限を削除するように修正する予定ですので、今しばらくお待ちください🙇

**Reply by yuta0709** [reply]
@tenmyo 
お待たせいたしました🙇

このたび、Gist連携サポートの全面終了に伴い新規のGitHub連携ではGistの権限を要求しないように変更いたしました。
詳細はこちらのブログをご確認ください。
https://blog.qiita.com/end-gist-coordination-support/

今後ともQiitaをよろしくお願いいたします！

**Comment by tenmyo** [comment]
QiitaはGitHubアカウントでログインしていました。
[記事](https://blog.qiita.com/end-gist-coordination-support/) の手順にあるように
GitHub側で Revoke し、 Qiitaで連携しなおす（ログアウト→GitHubアカウントでログインし連携）と
権限がAccess public information (read-only)
となることを確認しました。
ご対応ありがとうございました。
